content/protocols/sse.textile

```diff
@@ -0,0 +1,230 @@
+
+Read more in the "delta section":/channels/options/delta.
+
+h3(#delta-example). Delta example with SSE
+
+You can subscribe to messages in delta mode, using the SSE transport, as follows.
+
+```[javascript](code-editor:realtime/channel-deltas-sse-enveloped)
+  /* Make sure to include <script src="https://cdn.ably.com/lib/delta-codec.min-1.js"></script> in your head */
+  var key = '{{API_KEY}}';
+  var channel = '{{RANDOM_CHANNEL_NAME}}';
+  var baseUrl = 'https://realtime.ably.io/event-stream';
+  var urlParams = `?channels=${channel}&v=1.1&key=${key}&delta=vcdiff`;
+  var url = baseUrl + urlParams;
+  var eventSource = new EventSource(url);
+  var channelDecoder = new DeltaCodec.CheckedVcdiffDecoder();
+
+  eventSource.onmessage = function(event) {
+    /* event.data is JSON-encoded Ably Message
+       (see https://ably.com/docs/realtime/types#message) */
+    var message = JSON.parse(event.data);
+    var { id, extras } = message;
+    var { data } = message;
+
+    try {
+      if (extras && extras.delta) {
+        data = channelDecoder.applyBase64Delta(data, id, extras.delta.from).asUtf8String();
+      } else {
+        channelDecoder.setBase(data, id);
+      }
+    } catch(e) {
+      /* Delta decoder error */
+      console.log(e);
+    }
+
+    /* Process decoded data */
+    console.log(data);
+  };
+```
+
+h3(#delta-example-unenv). Delta example with unenveloped SSE
+
+For more information on enveloped and unenveloped SSE, please see the "SSE API":/api/sse#sse
+
+```[javascript](code-editor:realtime/channel-deltas-sse)
+  /* Make sure to include <script src="https://cdn.ably.com/lib/delta-codec.min-1.js"></script> in your head */
+  var DeltaCodec = require('@ably/delta-codec');
+
+  var key = '{{API_KEY}}';
+  var channel = 'sample-app-sse';
+  var baseUrl = 'https://realtime.ably.io/event-stream';
+  var urlParams = `?channels=${channel}&v=1.1&key=${key}&delta=vcdiff&enveloped=false`;
+  var url = baseUrl + urlParams;
+  var eventSource = new EventSource(url);
+  var channelDecoder = new DeltaCodec.VcdiffDecoder();
+
+  eventSource.onmessage = function(event) {
+      var data = event.data;
+
+      try {
+          if (DeltaCodec.VcdiffDecoder.isBase64Delta(data)) {
+              data = channelDecoder.applyBase64Delta(data).asUtf8String();
+          } else {
+              channelDecoder.setBase(data);
+          }
+      } catch(e) {
+          /* Delta decoder error */
+          console.log(e);
+      }
+
+      /* Process decoded data */
+      console.log(data);
+  };
+```
+
+h3(#rewind-sse). Rewind with SSE
+
+To use Rewind with SSE specify the @rewind@ channel option with the value @"1"@ using a querystring parameter, where it will apply to all channels.
```

Comment:
This makes it appear that you can only use rewind with `1` which isn't true. This is also contradictory as the following two examples show that you can apply it to all channels, or just one depending on how you specify the parameters. Also don't need to capitalise "rewind" here.